src/components/Modal/modal.scss

```diff
@@ -0,0 +1,7 @@
+@import "~vanilla-framework/scss/settings";
+
+.p-modal__button-row {
+  border-top: 1px solid $color-mid-light;
+  padding-top: 1rem;
```

Comment:
This could use `$spv-outer--small-scaleable`.

Reply:
Thanks, I'm not very up to date on these :)